src/Employer/Employer.Web/Orchestrators/Part1/WageOrchestrator.cs

```diff
@@ -31,17 +31,21 @@ public WageOrchestrator(IEmployerVacancyClient client, IRecruitVacancyClient vac
         public async Task<WageViewModel> GetWageViewModelAsync(VacancyRouteModel vrm)
         {
             var vacancy = await Utility.GetAuthorisedVacancyForEditAsync(_client, _vacancyClient, vrm, RouteNames.Wage_Get);
-            
+
+            var training = await _vacancyClient.GetApprenticeshipProgrammeAsync(vacancy.ProgrammeId);
+
             var vm = new WageViewModel
             {
                 Duration = vacancy.Wage?.Duration?.ToString(),
-                DurationUnit = vacancy.Wage?.DurationUnit ?? DurationUnit.Year,
+                DurationUnit = vacancy.Wage?.DurationUnit ?? DurationUnit.Month,
                 WorkingWeekDescription = vacancy.Wage?.WorkingWeekDescription,
                 WeeklyHours = $"{vacancy.Wage?.WeeklyHours:0.##}",
                 WageType = vacancy.Wage?.WageType,
                 FixedWageYearlyAmount = vacancy.Wage?.FixedWageYearlyAmount?.AsMoney(),
                 WageAdditionalInformation = vacancy.Wage?.WageAdditionalInformation,
-                PageInfo = Utility.GetPartOnePageInfo(vacancy)
+                PageInfo = Utility.GetPartOnePageInfo(vacancy),
+                TrainingTitle = training?.Title,
+                TrainingDurationMonths = training?.Duration ?? 0
```

Comment:
Shouldn't this be set depending on vacancy.Wage?.DurationUnit - because it might be years?

Reply:
The duration of training courses in FAT is always months so I think we're safe here.